packages/terra-list/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Unreleased
 ### Removed
 * Remove stylelint-disable selector-class-pattern
 
+### Changed
```

Comment:
Should this be :
###Added 
*SelectableList component

###Changed
*Update SingleSelectList and MultiSelectList components to use SelectableList